docs/faq.md

```diff
@@ -0,0 +1,21 @@
+# FAQ
+
+## When is it appropriate to use Ark instead of etcd's built in backup/restore?
+
+Etcd's restore tooling is good for recovering from data loss in a single etcd cluster, but does not
+support more sophisticated restores. If you only want to recover from data loss in a single etcd
+cluster, you're likely better off using etcd's backup/restore tooling.
```

Comment:
I still prefer avoiding saying "likely" here. I don't have the best way to phrase this in my head yet, but basically if your goal is to backup and restore your cluster _exactly as it was_, then you're really treating your cluster as a pet, which can make recovery more difficult (the node names have to be the same because they're recorded in pod.spec.nodeName, for example). We feel that Ark is a better approach because it gives you the ability to throw away an unstable cluster and restore your Kube resources & data into a new cluster.

Reply:
feel free to change it however you think is best! =)

Reply:
it might be good to say

> We feel that Ark is a better approach because it gives you the ability to throw away an unstable cluster and restore your Kube resources & data into a new cluster.

explicitly, that's an interesting perspective that I hadn't considered.

Reply:
I think it'd be a good idea to back up etcd prior to upgrading etcd itself, for example.

I'm going to merge this and do a follow-up.